descriptors/ModuleDescriptor-template.json

```diff
@@ -525,6 +525,17 @@
         }
       ]
     },
+    {
+      "id": "orders-storage.po_line_number",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/orders-storage/po-line-number",
+          "permissionsRequired": ["orders-storage.po_line_number.get"]
```

Comment:
1. Please use dash instead of underscore
2. Please add permission definition (permissionSets update)